README.md

```diff
@@ -13,15 +13,60 @@ This project is used as a dependency by several other projects.
 
 Proposals and issues for the specification itself should be submitted as pull requests to the [spec repo](https://github.com/decentralized-identity/decentralized-web-node).
 
-
 ## Installation
 
-If you are interested in using DWNs and web5 in your web app, you probably want to look at web5-js, instead of this repository. Head on over here: https://github.com/TBD54566975/web5-js
+If you are interested in using DWNs and web5 in your web app, you probably want to look at web5-js, instead of this repository. Head on over here: https://github.com/TBD54566975/web5-js.
+
+For advanced users wishing to use this repo directly:
 
 ```bash
 npm install @tbd54566975/dwn-sdk-js
 ```
 
+Additional steps needed for some environments.
+
+Node.js <= 18
+
+```js
+// node.js 18 and earlier,  needs globalThis.crypto polyfill
+import { webcrypto } from "node:crypto";
+// @ts-ignore
+if (!globalThis.crypto) globalThis.crypto = webcrypto;
+```
+
+Synchronous methods:
+
+```js
+// Enable synchronous methods.
+// Only async methods are available by default, to keep the library dependency-free.
+import { hmac } from "@noble/hashes/hmac";
+import { sha256 } from "@noble/hashes/sha256";
+import { sha512 } from "@noble/hashes/sha512";
+ed.etc.sha512Sync = (...m) => sha512(ed.etc.concatBytes(...m));
+secp.etc.hmacSha256Sync = (k, ...m) =>
+  hmac(sha256, k, secp.etc.concatBytes(...m));
+// Sync methods can be used now:
+// ed.getPublicKey(privKey); ed.sign(msg, privKey); ed.verify(signature, msg, pubKey);
+// secp.sign(msgHash, privKey);
+```
+
```

Comment:
Thanks @shamilovtim, this portion can probably be removed, because we don't use synchronous methods of the library.

Reply:
done